### ISSUE TITLE
on main branch: command yarn workspace "@dm3-org/dm3-messenger-web"  start:dev : It no longer works

### ISSUE THREAD

**Issue by enghibrahem** [dm3-org/dm3#1055]
I tried to run  **dm3-messenger-web** : so i execute commands yarn & yarn build and i add .env file and type command 
**yarn workspace "@dm3-org/dm3-messenger-web"  start:dev** 

but i faced this error :
![image](https://github.com/dm3-org/dm3/assets/167298173/2d41f1a7-32b5-4b34-acd1-9d08005784a3)
```
Uncaught TypeError: Cannot read properties of null (reading 'useMemo')
    at useMemo (react.development.js:1650:1)
    at Home (Home.js:38:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at beginWork$1 (react-dom.development.js:27426:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
    at renderRootSync (react-dom.development.js:26434:1)
    at recoverFromConcurrentError (react-dom.development.js:25850:1)
```
but when I  tried to run **dm3-messenger-widget** , using  command  **yarn workspace "@dm3-org/dm3-messenger-widget"  start:example** and saame .env file ,it run perfectly:
![image](https://github.com/dm3-org/dm3/assets/167298173/942500cf-2130-401a-8d27-1725568e18ac)
![image](https://github.com/dm3-org/dm3/assets/167298173/66b08896-c3c9-48f2-8925-b42a96cffd44)

**Comment by AlexNi245** [comment]
fixed by #1090

**Comment by AlexNi245** [comment]
Thanks for reporting! @enghibrahem
